exchanges/gateio/gateio_wrapper.go

```diff
@@ -1898,18 +1893,74 @@ func (g *Gateio) GetHistoricCandles(ctx context.Context, pair currency.Pair, a a
 				Volume: candles[x].Volume,
 			}
 		}
-	case asset.Options:
```

Comment:
gateio_test.go:2523: GateIO GetHistoricCandles() expecting: not yet implemented, but found unsupported asset type asset type: options